modules/ROOT/pages/usage-and-engagement-metrics.adoc

```diff
@@ -0,0 +1,40 @@
+= Usage and Engagement Metrics
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Exchange provides usage and engagement metrics for administrators and asset owners to gain insight into the consumption and user engagement of their API specification fragments and REST API assets. 
+
+The usage metrics measure:
+
+* The number of times the asset was downloaded from Exchange.
+* The number of imports from API Designer.
+* The number of imports from Anypoint Studio.
+* The number of dependents for the asset.
+* The number of contracts for the asset. 
+
+The asset engagement score is a composite metric that administrators and asset owners can use to measure the usage of the assets in Exchange. 
+
+Administrators can view the *Usage and engagement metrics* dashboard for metrics of API specification fragments and REST API assets in an organization. Administrators and asset owners with the Exchange contributor permission can view usage and engagement metrics for a specific asset version from the asset details page. You can share data by exporting metrics to a CSV file.
+
+Users with starter and industry license packages cannot access the usage and engagement metrics.
+
+Exchange assets capture usage and engagement metrics to:
+
+* Drive growth, engagement, and reuse across the entire catalog.
```

Comment:
```suggestion
* Drive growth and engagement, and reuse across the entire catalog.
```

Reply:
What can be reused? Can we say "and to reuse assets" 
I don't know if that's accurate, so you can decide on that one.

Reply:
Yes, I will change it to "reuse assets for clarity.